data-service/src/main/java/com/amazonaws/blox/dataservice/repository/EnvironmentRepositoryDDB.java

```diff
@@ -308,15 +312,23 @@ public Environment getEnvironment(@NonNull final EnvironmentId environmentId)
   }
 
   @Override
-  public List<Environment> listEnvironments(@NonNull final Cluster cluster)
+  public List<Environment> listEnvironments(
```

Comment:
You may keep the original method to have the helper method to list all Enviornment without prefix, such as 
```
public List<Environment> listEnvironments(@NonNull final Cluster cluster) {
  this(....);
}
```

Reply:
Yeah, good catch!